definition-map.js

```diff
@@ -75,7 +75,56 @@ const QUERIES = {
             includeCorrectAnswers: "Flag to include questions the user previously got correct.",
             quizzes: "The IDs of the Quizzes user selects."
         }
-    }
+    },
+    Forums: {
+        metadescription: "This query returns a forum object. A forum is any area you can add a discussion to. For example, assignments, forum page type, discussions below the course on each section.",
+        args: {
+            courseId: "The ID of the course. This arguement is required", 
```

Comment:
Don't need to say the field is required here, should be evident via argument name field when it appears in the rendered documentation, e.g.

```
courseId - ID!

vs

courseId - ID
```